sdk/Lusid.Sdk.Tests/Utilities/TestDataUtilities.cs

```diff
@@ -304,18 +318,18 @@ public void UpsertCdsSpreadCurves(string scope, DateTimeOffset effectiveAt, stri
             // CREATE upsert equity instrument request, if equityIdentifier is provided
             if (!equityIdentifier.IsNullOrEmpty())
```

Comment:
what is special about equity identifiers? i.e. how does a new user know when to use this field vs. standard instrument identifiers?

Reply:
I'm not sure I didn't write this code. It looks like this is never used from the tests - will see if I can unify this to have one generic upsert instrument method, but inclined to leave as a lower priority at this stage.